test/integration/component/test_acl_listvolume.py

```diff
@@ -357,9 +357,9 @@ def tearDownClass(cls):
         cls.apiclient = super(TestVolumeList, cls).getClsTestClient().getApiClient()
         cls.apiclient.connection.apiKey = cls.default_apikey
         cls.apiclient.connection.securityKey = cls.default_secretkey
+        cleanup_resources(cls.apiclient, cls.cleanup)
```

Comment:
Disregard, I can't read the difference between `test_acl_listvolume.py` and `test_acl_listvm.py`.  Deleting both of these comments.  This post is for everyone who follows this stuff in their email and are confused...